### ISSUE TITLE
Additional Safe Area Height in iPhone X

### ISSUE THREAD

**Issue by MrJai** [iosphere/ISHPullUp#76]
Hi, thanks for such an amazing view controller that saved me many hours of work, and it works great. I just face one issue recently and that is specific to iPhone X.

There is additional 34 pixel at the bottom of screen when running on iPhone X. You can see the same in the sample maps app. There should be an option to choose from if a developer wants bottom of super view to be the reference or bottom of safe area to be the reference.

If you think there can be a way around to fix it, then please help me out.

**Comment by matthewharries** [comment]
I agree! For now I just did the following to calculate my height `return tableView.contentSize.height + (UIApplication.shared.keyWindow?.safeAreaInsets.bottom ?? 0)`
